cpp/src/parquet/column_reader_test.cc

```diff
@@ -26,6 +27,7 @@
 #include <utility>
 #include <vector>
 
+#include "arrow/api.h"
```

Comment:
`arrow/api.h` will include too much, can you include only the required concrete headers instead?

Reply:
Done.